### ISSUE TITLE
Textarea element is hidden in form if the element was hidden field before

### ISSUE THREAD

**Issue by robho** [joomlahenk/fabrik#605]
**To Reproduce**

1. Create a form with a "field" element.
2. Set the element to "Hidden".
3. Change the element to a "textarea" element.
4. Now the textarea element is hidden in the form and you have to convert to element back to a field element in order to set hidden = false.

**Expected behavior**
Either
- The hidden attribute should be toggleable for other element types than field
or
- The hidden attribute should only be used when the element is of type field

**Actual behavior**
The textarea element is hidden in the form view and there's no way to "unhide" unless the element is converted to type field.

**Comment by trob** [comment]
This is the same behaviour as in Fabrik3.

**Comment by skurvish** [comment]
Yes, changing element types messes with the params. Some params may have equivalent properties between fields, yet others properties do not. The issue is if you change an element for field to textarea, and we wipe the values for any that do not apply, then switch back to field, how is this to be handled?

My opinion is that if you are going to change the element type, delete the existing element and create a new one.

**Comment by trob** [comment]
"Hidden" is special as it's a column in #_fabrik_elements, not in params. So the switch could be added without messing params.
But as I said it's only "nice to have".

**Comment by dimoss** [comment]
Because of this switch lack I was handling similar cases directly in the DB in #_fabrik_elements changing 'hidden' value to 0/1.
